### PR TITLE
Admin tab translation

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by OskarStark** [comment]
LGTM

File: Resources/views/Core/tab_menu_template.html.twig [hunk 1]
```diff
@@ -115,4 +115,11 @@
     </a>
 {% endblock %}
 
-{% block label %}{{ item.label|trans }}{% endblock %}
+{% block label %}
+{{-
+    item.label|trans(
+        item.getExtra('translation_params', {}),
+        item.getExtra('translation_domain', item.getParent('translation_domain'))
```
Comment (Koc) [inline Resources/views/Core/tab_menu_template.html.twig]: maybe `item.getParent() ? item.getParent().getExtra('translation_domain') : null` ?

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: It is the same, the second parameter of `getExtra` is the default value, apparently.

Comment (Koc) [inline Resources/views/Core/tab_menu_template.html.twig]: I know that. Question about `getParent` hasn't argument and looks like you should get extra from parent.

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: Oh, you mean if item has no parent, right? I use this code in production, so I guess it must return null, but let me check.

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: The interface says so : 

```php
    /**
     * @return ItemInterface|null
     */
    public function getParent();
```

and the implementation is simple : 

```php
    public function getParent()
    {
        return $this->parent;
    }
```

Is this ok for you?

Comment (Koc) [inline Resources/views/Core/tab_menu_template.html.twig]: You missunderstand me. Do you want get translation domain from parent if no translation domain on item?

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: Yes exactly.

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: Oh ok I finally got what you mean. I must be very tired.

Comment (greg0ire) [inline Resources/views/Core/tab_menu_template.html.twig]: fixed @Koc

**Comment by greg0ire** [comment]
It would be great to have integration tests for that, I'll see what I can do.

**Comment by greg0ire** [comment]
Managed to write some tests, but they fail with `--prefer-lowest`. Trying to figure out why.

**Comment by greg0ire** [comment]
Ok so this is the commit that changes the way I should test: https://github.com/symfony/symfony/commit/ccd32d599584eb75ef049e5c4a03a2f265d831ba

**Comment by greg0ire** [comment]
Yay! It works! Ping @OskarStark @core23

**Comment by core23** [comment]
Thank you @greg0ire

**Pull request merged**